src/components/TitleHashButton.tsx

```diff
@@ -135,13 +136,24 @@ function Name({address, isValidator}: {address: string; isValidator: boolean}) {
           padding: "0.15rem 1rem 0.15rem 1rem",
         }}
       >
-        <Link
-          href={`https://www.aptosnames.com/name/${name}`}
-          target="_blank"
-          underline="none"
-        >
-          {`${name}.apt`}
-        </Link>
+        {name.endsWith(".apt") ? (
```

Comment:
I _think_ we have special treatment for `.petra` names as well. To be safe, change this to:
```ts
const isAns = name.endsWith(".apt") || name.endsWith(".petra")
{isAns ? () : ()}
```